### DISCUSSION TITLE
我无法使用入库配置工具 我想手动添加游戏 但咱们这个项目里面好像没有密匙?

### DISCUSSION THREAD

**Discussion by HUMHIHello** [pjy612/SteamManifestCache#69] [Question / Answer] [answered]
如题
我的入库配置工具会报错 
这个项目里面有我想要的游戏 327860
而heyong项目中没有
很尴尬的是 咱们这个项目里面有清单 但是没有密匙?
config.json 与 appinfo.vdf 是做什么的?
我不知道这俩个文件如何使用

**Comment by SunshnX** [comment, marked as answer]
由于倒卖猖獗，所以 共享密钥 改为 内测使用。
欢迎在 github 分享密钥和清单，成功分享能加入内测。

**Reply by HUMHIHello** [reply]
谢了 分享了三个了 也骚扰管理员三次了 希望不要骂我 qwq

**Reply by HUMHIHello** [reply]
我看电报里面的说 每分享一次你们都会被提示

**Reply by ZEERDEER** [reply]
不是，关闭通知就可以了

**Comment by BaeC2004** [comment]
请问密钥如何查看？

**Comment by Nobody2613** [comment]
感谢对项目的贡献！
密钥已发送至你的邮箱
可通过GreenLuma或邮箱内提供的另一种思路进行下载

**Reply by HUMHIHello** [reply]
谢谢您了

**Comment by EricDasha** [comment]
我好像遇到熟人了😐
